shared/chat/conversation/messages/cards/team-journey/index.tsx

```diff
@@ -118,9 +119,14 @@ const styles = Styles.styleSheetCreate(
         left: '50%',
         position: 'absolute',
```

Comment:
What does this look like on an iPhone SE? Will it clip behind the buttons/text, especially if it's the 'join these other channels' card.

Reply:
Good question, not able to test on iPhone SE right now 'cause the kb_cards_* channel names for testing in the service stopped working.  Will check once they're fixed!